website/content/en/preview/concepts/nodeclaims.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "NodeClaims"
+linkTitle: "NodeClaims"
+weight: 30
+description: >
+  Understand NodeClaims
+---
+
+Karpenter uses NodeClaims to manage the lifecycle of Kubernetes Nodes with the underlying cloud provider.
+Karpenter's algorithm creates and deletes NodeClaims in response to the demands of the Pods in the cluster.
+NodeClaims are created from the requirements of existing [NodePool]({{<ref "./nodepools" >}}) and associated
+[NodeClasses]({{<ref "./nodeclasses" >}}).
+While NodeClaims require no direct user input, as a Karpenter user you can monitor NodeClaims to keep track of
+the status of your nodes in cases where something goes wrong or a node drifts from its intended state.
+
+Karpenter uses NodeClaims to request capacity and provide a running representation of that capacity.
+Karpenter creates NodeClaims in response to provisioning and disruption needs (pre-spin). Whenever Karpenter
+creates a NodeClaim, it asks the cloud provider to create the instance (launch), register the created node
+with the node claim (registration), and wait for the node and its resources to be ready (initialization).
+
+This page describes how NodeClaims relate to other components of Karpenter and the related cloud provider.
+
+If you want to learn more about the nodes being managed by Karpenter, depending on what you are interested in,
```

Comment:
It might be worth adding an example of a NodeClaims, just like we do for the nodepool and nodeclass